### ISSUE TITLE
Support DD MM SS[optional comma] DD MM SS coordinates format in the search

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#8228]
Looks like these coordinates are read by Google Maps:
`00 20 31 ,00 51 29`
`29 51 00, 31 20 00`

It should be easy to parse these in OM too.

**Comment by greg-ware** [comment]
So these would be the corners of a search 'rectangle', right?

**Comment by biodranik** [comment]
No, it's a single coordinate search. Above are 4 different coordinates.

**Comment by ammonia2** [comment]
Hi, I'd like to work on this issue.

I've examined `search/latlon_match.cpp`, specifically the `EatDouble` function within `MatchLatLonDegree`. This function seems to handle both '.' and ',' as decimal separators. It appears to only replace a comma with a period.

Is this issue still open?
